internal/security/bootstrapper/command/setupacl/command.go

```diff
@@ -188,6 +204,110 @@ func (c *cmd) GetCommandName() string {
 	return CommandName
 }
 
+// createEdgeXACLTokenRoles creates secret store roles that can be used for genearting registry tokens
+// via Consul secret engine API /consul/creds/[role_name] later on for all EdgeX microservices
+func (c *cmd) createEdgeXACLTokenRoles(secretstoreToken string, rolePolicy Policy) error {
+	roleNames := c.configuration.StageGate.Registry.ACL.GetACLRoleNames()
+	if len(roleNames) == 0 {
+		c.loggingClient.Warn("found no ACL role names defined in configuration, skip create ACL roles")
+	}
```

Comment:
`return nil` here? I know it result is the same since for loop does nothing, but for the reader it makes sense per the message.

Reply:
fixed